src/dist/temp.rs

```diff
@@ -27,10 +28,11 @@ pub enum Notification<'a> {
     DirectoryDeletion(&'a Path, io::Result<()>),
 }
 
+#[derive(Clone)]
 pub struct Cfg {
     root_directory: PathBuf,
     pub dist_server: String,
-    notify_handler: Box<dyn Fn(Notification<'_>)>,
+    notify_handler: Arc<dyn Fn(Notification<'_>)>,
```

Comment:
I'm really not happy about all this incidental noise from making `Cfg` derive `Clone`

Especially how it results in noise in the dist tests too :(